### PR TITLE
added german as language

### PR DESCRIPTION
**Pull request by bl00dy1837**

### CODE REVIEW

File: assets/l10n/strings.json [hunk 1]
```diff
@@ -1,170 +1,212 @@
 {
 	"start_game": {
 		"en": "Start Game",
-		"it": "Nuova Partita"
+		"it": "Nuova Partitai",
```
Comment (silverweed) [inline assets/l10n/strings.json]: There is a stray 'i' at the end of the italian version, please remove it

**Review by silverweed — CHANGES_REQUESTED** [review]
Many thanks for the contribution!
I just spotted a minor mistake, if you could please fix it I'll be happy to merge :)

**Review by silverweed — APPROVED** [review]

**Comment by silverweed** [comment]
Thanks again, I merged the changes :)
I'll try to rebuild the game and make a new release as soon as I can (I should really set up CI to do that automatically...)

**Comment by bl00dy1837** [comment]
You’re welcome.
Sorry for the "i" it was late, vim typo.  :)
Like the game. Thx for it

**Comment by silverweed** [comment]
Btw I fixed the layout of the German win screen by changing a couple of newlines (as some lines were too long).
I also noticed that a couple of strings weren't localized so I fixed that too (I hope "du hast gewonnen" is a good translation for "you have won" - otherwise please tell me a better one :D)

**Comment by bl00dy1837** [comment]
"Du hast gewonnen" is perfect. :) thx

**Pull request merged**